concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -0,0 +1,694 @@
+Teit-1908-693-626	626	570	bridge	1011	BRIDGE	86
+Teit-1908-693-627	627	571	train	900	TRAIN	86
+Teit-1908-693-628	628	572	steamboat			86
+Teit-1908-693-629	629	573	telegraph			86
+Teit-1908-693-630	630	574	telephone	1379	TELEPHONE	86
+Teit-1908-693-631	631	575	interpreter	3744	INTERPRETER	86
+Teit-1908-693-632	632	576	I	1209	I	86
+Teit-1908-693-633	633	577	thou	1215	THOU	86
+Teit-1908-693-634	634	578	he she it	262	HE OR SHE OR IT	86
+Teit-1908-693-635	635	579	we	1212	WE	86
+Teit-1908-693-636	636	580	you	1213	YOU	86
+Teit-1908-693-637	637	581	they	817	THEY	86
+Teit-1908-693-638	638	582	to go outside			86
+Teit-1908-693-639	639	583	to go inside			86
+Teit-1908-693-640	640	584	to strike with fist	353	STRIKE	86
+Teit-1908-693-641	641	585	to kick	1135	KICK	88
+Teit-1908-693-642	642	586	to drink	1401	DRINK	88
+Teit-1908-693-643	643	587	to eat	1336	EAT	88
+Teit-1908-693-644	644	588	to talk	2708	TALK	88
+Teit-1908-693-645	645	589	to cry	1839	CRY	88
+Teit-1908-693-646	646	590	to sing	1261	SING	88
+Teit-1908-693-647	647	591	to laugh	1355	LAUGH	88
+Teit-1908-693-648	648	592	to walk	1443	WALK	88
+Teit-1908-693-649	649	593	to dance (general name)	1879	DANCE	88
+Teit-1908-693-650	650	594	to hunt	1435	HUNT	88
+Teit-1908-693-651	651	595	to go	695	GO	88
+Teit-1908-693-652	652	596	to go home			88
+Teit-1908-693-653	653	597	to come	1446	COME	88
+Teit-1908-693-654	654	598	 to run	1519	RUN	88
+Teit-1908-693-655	655	599	to swim	1439	SWIM	88
+Teit-1908-693-656	656	600	to swim across			88
+Teit-1908-693-657	657	601	to cross water			88
+Teit-1908-693-658	658	602	to reach over water to opposite side			88
+Teit-1908-693-659	659	603	to go downstream canoe			88
+Teit-1908-693-660	660	604	to go upstream canoe			88
+Teit-1908-693-661	661	605	hungry	1533	BE HUNGRY	88
+Teit-1908-693-662	662	606	thirsty	847	THIRSTY	88
+Teit-1908-693-663	663	607	good	1035	GOOD	88
+Teit-1908-693-664	664	608	bad	1292	BAD	88
+Teit-1908-693-665	665	609	cold	1287	COLD	88
+Teit-1908-693-666	666	610	warm	1232	WARM	90
+Teit-1908-693-667	667	611	which	727	WHICH	90
+Teit-1908-693-668	668	612	where	1237	WHERE	90
+Teit-1908-693-669	669	613	why	1270	WHY	90
+Teit-1908-693-670	670	614	what	1236	WHAT	90
+Teit-1908-693-671	671	615	who	1235	WHO	90
+Teit-1908-693-672	672		plural [who]			90
+Teit-1908-693-673	673	616	high	1265	HIGH	90
+Teit-1908-693-674	674	617	low	721	LOW	90
+Teit-1908-693-675	675	618	up	1591	UP	90
+Teit-1908-693-676	676	619	down	1326	DOWN	90
+Teit-1908-693-677	677	620	across	2670	ACROSS	90
+Teit-1908-693-678	678	621	tall	711	TALL	90
+Teit-1908-693-679	679	622	large	1202	BIG	90
+Teit-1908-693-680	680	623	small	1246	SMALL	90
+Teit-1908-693-681	681	624	long	1203	LONG	90
+Teit-1908-693-682	682	625	short	1645	SHORT	90
+Teit-1908-693-683	683	626	strong	785	STRONG	90
+Teit-1908-693-684	684	627	yes	1268	YES	90
+Teit-1908-693-685	685	628	no	1269	NO	90
+Teit-1908-693-686	686	629	thank you			90
+Teit-1908-693-687	687	630	come here	3555	COME HERE	90
+Teit-1908-693-688	688	631	go away			90
```

Comment:
1752 LEAVE ?

Reply:
I left it unmapped because it looks like an imperative, judging by the concepts right above.

Reply:
Ah yes, then we should leave it unmapped.